### PR TITLE
docs: Initial proposal of the airline data structure

### PR DESCRIPTION
**Pull request by hynek-urban**
This PR introduces the initial proposal for the WT airline data structure. The naming convention is inspired by the Open Travel standard (https://schemas.liquid-technologies.com/OpenTravel/2008B/).  Any kind of feedback is welcome.

Once it is merged, we can update the wiki a little to cover airlines alongside hotels in a more systematic way. Together with that, we can prepare the proposition for consultation with our airline mentors.

There is one major thing I suspect cannot stay the way it is in the proposal now, maybe not even temporarily - namely, the prices. Currently, every flight segment has a separate fare with the implicit assumption that the price for a journey that covers multiple flight segments is computed as a sum of the individual flight segment prices. Almost certainly, though, this is not how pricing works in practice.

Aside from that, we are aware that we have "hand-waived" a lot of other pricing problems away for the moment. Maybe it would be good to consider whether we shouldn't take a "hybrid" approach and instead of containing the price information directly in the data structure, hide them beind a "pricingUri" following a similar principle to the existing "bookingUri" (i.e., limit our scope to defining an interface for finding the price for a given query).

### CODE REVIEW

File: airlines-data-swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+          description: Ethereum address of the airline
+          $ref: 'https://raw.githubusercontent.com/windingtree/wiki/2c6312c06b15a2c12fe764ce10bdb6c4a0f78843/shared-resources.yaml#/components/schemas/EthereumAddressType'
+        manager:
+          description: Ethereum address of the manager wallet
+          $ref: 'https://raw.githubusercontent.com/windingtree/wiki/2c6312c06b15a2c12fe764ce10bdb6c4a0f78843/shared-resources.yaml#/components/schemas/EthereumAddressType'
+        dataUri:
+          description: URI of the Airline data index (pointing off-chain)
+          $ref: 'https://raw.githubusercontent.com/windingtree/wiki/2c6312c06b15a2c12fe764ce10bdb6c4a0f78843/shared-resources.yaml#/components/schemas/UriType'
+        created:
+          description: Number of a block when the hotel was created
```
Comment (JirkaChadima) [inline airlines-data-swagger.yaml]: `hotel` -> `airline`

File: airlines-data-swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+          format: date-time
+    Flight:
+      type: object
+      description: Data for a specific flight.
+      required:
+        - dateTime
+        - segments
+        - updatedAt
+      properties:
+        dateTime:
```
Comment (JirkaChadima) [inline airlines-data-swagger.yaml]: Wouldn't `departureDateTime` be more descriptive? `dateTime` by itself could mean virtually anything.

File: airlines-data-swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+              - departureDateTime
+              - arrivalDateTime
+              - bookingClasses
+            properties:
+              departureDateTime:
+                description: Time of the departure.
+                type: string
+                format: date-time
+              arrivalDateTime:
+                description: Time of the arrival.
```
Comment (JirkaChadima) [inline airlines-data-swagger.yaml]: How do these behave when a single flight spans across multiple timezones? I suppose that you should declare all the date times with timezones, so maybe we can just explicitely say "with timezone" here.

File: airlines-data-swagger.yaml (hunk 1 above)
Comment (hynek-urban) [reply airlines-data-swagger.yaml]: Done.

File: airlines-data-swagger.yaml (hunk 2 above)
Comment (hynek-urban) [reply airlines-data-swagger.yaml]: Good point, done.

File: airlines-data-swagger.yaml (hunk 3 above)
Comment (hynek-urban) [reply airlines-data-swagger.yaml]: Timezone is already requested by the "date-time" format, which is specified by the OpenAPI standard so I believe we're covered :)

**Comment by hynek-urban** [comment]
I'm closing this for now because a Winding Tree Improvement Proposal should be created instead.

**Pull request closed without merging**